File: timetable_assistant.py
import datetime
import locale

# 時間割マスタ
timetable = {
    "Mon" : ("体育", "国語", "数学", "英語", "物理"),
    "Tue" : ("体育", "国語", "数学", "英語", "物理"),
    "Wed" : ("体育", "国語", "数学", "英語", "物理"),
    "Thr" : ("体育", "国語", "数学", "英語", "物理"),
    "Fri" : ("体育", "国語", "数学", "英語", "物理"),
    "Sat" : ("体育", "国語", "数学", "英語", "物理"),
    "Sun" : ("体育", "国語", "数学", "英語", "物理")
}

# 今日の曜日を出力
today = datetime.date.today()
day_of_week = today.strftime("%a")

print(day_of_week)
print("本日の時間割りは")
for lesson in timetable[day_of_week]:
    print(lesson)
print("です。")

for day in range(7):
    print((today + datetime.timedelta(days=day)).strftime("%a"))

# ロケールを設定
locale.setlocale(locale.LC_ALL, "ja_JP.UTF-8")
print(today.strftime("%a"))

for day in range(7):
    print((today + datetime.timedelta(days=day)).strftime("%a"))

print(range(7))

File: timetable_assistant.js
const moment = require("moment");
const player = require("./WavPlayer")

const timetable = {
    "0" : ["お休み"], // 日曜日
    "1" : ["体育", "国語", "数学", "英語", "物理"],
    "2" : ["体育", "国語", "数学", "英語", "物理"],
    "3" : ["体育", "国語", "数学", "英語", "物理"],
    "4" : ["体育", "国語", "数学", "英語", "物理"],
    "5" : ["体育", "国語", "数学", "英語", "物理"],
    "6" : ["お休み"] // 土曜日
};

const today = moment();
console.log(today);

player.play("./時間割音声/本日の時間割は.wav")
for (lesson of timetable[today.day()]) {
    console.log(lesson);
}
console.log("です。")